packages/airnode-deployer/src/cli/index.ts

```diff
@@ -118,58 +118,24 @@ yargs(hideBin(process.argv))
     }
   )
   .command(
-    'remove-with-deployment-details',
-    'Removes a deployed Airnode instance using the Airnode short address and cloud provider specifications',
-    {
-      'airnode-address': {
-        alias: 'a',
-        description: 'Airnode Address',
-        type: 'string',
-        demandOption: true,
-      },
-      stage: {
-        alias: 's',
-        description: 'Stage (environment)',
-        type: 'string',
-        demandOption: true,
-      },
-      'cloud-provider': {
-        alias: 'c',
-        description: 'Cloud provider',
-        choices: availableCloudProviders,
-        demandOption: true,
-      },
-      region: {
-        alias: 'e',
-        description: 'Region',
+    'remove <deployment-id>',
```

Comment:
Btw. Can the deployment ID be found on the cloud provider? (I know it doesn't matter much, since the intended flow is to use the deployer to list the deployments)

Reply:
No, it can't. That's (partially) why I suggested doing https://github.com/api3dao/airnode/issues/1486 But as you said, shouldn't matter much anyway.